crud/src/main/java/com/redhat/lightblue/crud/CRUDController.java

```diff
@@ -125,4 +120,9 @@ CRUDFindResponse find(CRUDOperationContext ctx,
      * The back end should update the predefined fields of the document
      */
     void updatePredefinedFields(CRUDOperationContext ctx, JsonDoc doc);
+    
+    /**
+     * Checks the CRUD health
```

Comment:
Maybe make comment more meaningful or remove it. If a comment is literally the same thing as the method name it's just a waste.